src/types/upload.rs

```diff
@@ -111,6 +111,7 @@ impl Type for Upload {
             description: None,
             is_valid: |value| matches!(value, Value::String(_)),
             visible: None,
+            specified_by_url: None,
```

Comment:
Maybe we could add the multipart spec? `https://github.com/jaydenseric/graphql-multipart-request-spec`

Reply:
> Maybe we could add the multipart spec? `https://github.com/jaydenseric/graphql-multipart-request-spec`

done🙂